ui-tests/src/test/java/org/openmrs/reference/CheckModulesTest.java

```diff
@@ -28,7 +30,7 @@ public void checkModulesTest() {
         for (WebElement eachModule : firstColumn) {
             // The name attr on the <input> elements should all be "stop" which indicates the module is correctly started.
             // If not, then grab the text from the 3rd column to show which module is not started.
-            Assert.assertEquals("module not ready: " + eachModule.findElement(By.xpath("../../td[3]")).getText(), "stop", eachModule.getAttribute("name"));
+            assertFalse(eachModule.getText().contains("moduleNotStarted"));
```

Comment:
Is it moduleNotStarted or **NotStarted** in your assertion

Reply:
@sherrif10   contains looks for `moduleNotStarted` bt as well also `NotStarted` works...... bt `moduleNotStarted` is better how do u see



![Screenshot (18)](https://user-images.githubusercontent.com/86114366/137698037-6cb1d44b-2fa9-4fb7-aff1-7a8d866ebc89.png)